### ISSUE TITLE
Download Location

### ISSUE THREAD

**Issue by moritztim** [Oakchris1955/another-youtube-downloader#2]
A separate dir for downloads would be nice, so they don't end up in the main dir. Or maybe just a `../`, cause that would be consistant with your advice to download the files in the dir you want the video

**Comment by Oakchris1955** [comment]
It is just beta, calm down

**Comment by moritztim** [comment]
Bruh the point of a public beta is feedback

**Comment by Oakchris1955** [comment]
Yes, but this is the first commit. If you wanna help make a pull request. Also, use your real account plz

**Comment by Oakchris1955** [comment]
> A separate dir for downloads would be nice, so they don't end up in the main dir.

Well, I will reopen this and maybe have the download location be `Downloads/another-vid-downloader/video_name`

**Comment by Oakchris1955** [comment]
I am programming Beta v2 right now, it will succeed Beta v1 and will have (temporarily) the download location to download folder
